### PR TITLE
Feature final classes

### PR DESCRIPTION
**Pull request by nsobadjiev**
# Introduction

Added `final` to all classes in the SDK and sample app that shouldn't have to be subclassed.

# Merge info

Merge after #101 !!!

### CODE REVIEW

**Comment by EarlOfEgo** [comment]
```
Test Suite 'GiniSwitchSDK_Tests.xctest' failed at 2018-01-31 09:08:39.361.

	 Executed 203 tests, with 101 failures (101 unexpected) in 4.265 (4.392) seconds
```

File: GiniSwitchSDK/GiniSwitchSDK/Classes/OnboardingPageViewController.swift [hunk 1]
```diff
@@ -10,8 +10,8 @@ import UIKit
 
 class OnboardingPageViewController: UIViewController {
```
Comment (kikettas) [inline GiniSwitchSDK/GiniSwitchSDK/Classes/OnboardingPageViewController.swift:11]: You missed this one.

**Review by kikettas — APPROVED** [review]
👍. Just add the `final` to the `OnboardingPageViewController`

File: GiniSwitchSDK/GiniSwitchSDK/Classes/OnboardingPageViewController.swift (hunk 1 above)
Comment (nsobadjiev) [reply GiniSwitchSDK/GiniSwitchSDK/Classes/OnboardingPageViewController.swift:11]: I thought about and but decided not to make it final. It's a long shot, but theoretically, if someone what's to have a different kind of onboarding page, they might decide to subclass this one.

**Pull request merged**